container_pipeline/workers/build.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+import hashlib
+import json
+import logging
+import os
+
+from container_pipeline.utils import Build, get_job_name
+from container_pipeline.lib.log import load_logger
+from container_pipeline.lib.openshift import Openshift, OpenshiftError
+from container_pipeline.workers.base import BaseWorker
+
+
+class BuildWorker(BaseWorker):
+    """Build worker"""
+
+    def __init__(self, logger=None, sub=None, pub=None):
+        super(BuildWorker, self).__init__(logger, sub, pub)
+        self.openshift = Openshift(logger=self.logger)
+
+    def handle_job(self, job):
+        """Handle job"""
+        parent_build_running = False
+        parents = job.get('depends_on', '').split(',')
+        parents_in_build = []
+        for parent in parents:
+            is_build_running = Build(parent).is_running()
```

Comment:
> also if we can may be we can use the TEST_TAG here. as it is available through out the system, and we are creating dir for this in the beginning.

@bamachrn you're right about `TEST_TAG`'s availability across the system. However, a directory created using `TEST_TAG` doesn't get deleted from NFS. That's probably not a good way to judge if a parent image is building *right now*. This is probably a use case which can leverage redis/rabbitmq, maybe?